remember-remember.py

```diff
@@ -0,0 +1,20 @@
+from PIL import Image
+
+
+def decrypt_image(path: str):
+    """
+    This function decrypted message from the image (if exit one)
+    :param path: Get a path of a image encrypted.
+    """
+    # importing PIL
+    img = Image.open(path)
```

Comment:
Seprate IO from logic - the prints and opening the file from the logic of the function 
https://wiki.c2.com/?SeparateIoFromCalculation